docs/schemas/domain/IfcRailDomain/Types/IfcRailwayPartTypeEnum.md

```diff
@@ -2,56 +2,83 @@
 
-TRACKSTRUCTURE could have longitudinal decomposition based on track specificities:
+Figure 1 &mdash; Example of usage of _IfcRailwayPart_ to vertically organise the elements of a railway line
 
-* PLAINTRACKSUPESTRUCTURE for the plain line tracks (_IfcElementAssemblyTypeEnum_ TRACKPANEL).
-* DILATATIONSUPERSTRUCTURE for the area of dilatation panels (_IfcElementAssemblyTypeEnum_ DILATATIONPANEL).
-* TURNOUTSUPERSTRUCTURE for the area of turnouts (_IfcElementAssemblyTypeEnum_ TURNOUTPANEL).
-* TRACKSTRUCTUREPART for generic longitudinal decomposition if needed.
+> NOTE  If required by the use case, the part below the track can be distinguished between _IfcRailwayFacilityPart.SUBSTRUCTURE_ (for constructed ground) and _IfcFacilityPartCommon.BELOWGROUND_ (for non constructed, existing ground).
 
-![Railway vertical and lateral spatial decomposition ](../../../../figures/IfcRailwayPartTypeEnum-track.png )
+**Detailed vertical organisation**
+If required by the use case, the *SUBSTRUCTURE* part can contain elements such as _IfcCourse_ or _IfcEarthworksFill_ to distinguish the different substructure layers. An example is captured in the Figure below.
 
-Figure 2 &mdash; Track longitudinal decomposition
+![Railway part detailed vertical organisation](../../../../figures/IfcRailwayPartTypeEnum-substructure.png)
 
-More generic longitudinal subdivision is provided for LINESIDESTRUCTURE with LINESIDESTRUCTUREPART value if needed.
+Figure 2 &mdash; Example of the courses and elements that may be contained in the *SUBSTRUCTURE* part 
+
+**Longitudinal organisation**
+
+Tracks may have further longitudinal organisation based on track specificities:
+
+* PLAINTRACK for the plain line tracks (may contain _IfcElementAssembly.TRACKPANEL_).
+* TURNOUTTRACK for the area of turnouts (may contain _IfcElementAssembly.TURNOUTPANEL_).
+
+As depicted in the example Figure below. Other parts, not included in the example, may be:
+* DILATATIONTRACK for the area of dilatation panels (may contain _IfcElementAssembly.DILATATIONPANEL_).
+* TRACKPART for generic-purpose longitudinal organisation of parts of the track.
+
+![Railway part longitudinal organisation](../../../../figures/IfcRailwayPartTypeEnum-track.png)
+
+Figure 3 &mdash; Example of _IfcRailwayPart_ to longitudinally organise the elements of a railway line
+
+**Mixed organisation**
+The attribute _IfcFacilityPart.UsageType_ allows to to spatially organise the same dataset using multiple criteria. For example, a general vertical organisation, like in Figure 1, and a further longitudinal organisation just for the track, as in Figure 2. 
+
+> NOTE  This creates an implicit hierarchy among spatial parts and increases the complexity of the model.
 
 ## Items
 
-### TRACKSTRUCTURE
-A spatial structure element that contains track-related elements.
+### TRACK
+A spatial part that contains track-related elements, for example rails and sleepers.
```

Comment:
Is the a reason why you keep referring to "spatial part"? This is not common IFC lingo, where "spatial structure element" is.

Reply:
fixed